docs/indexer/custom-processors/e2e-tutorial.md

```diff
@@ -73,107 +64,113 @@ For mac, if you're using brew, start it up with:
     - rust
     - scripts
     - typescript
 ```
 
-Once you have your config.yaml file open, you only need to change one field, if you just want to run the processor as is:
-
+Once you have your config.yaml file open, you only need to update a certain fields
 ```yaml
-grpc_data_stream_api_key: "<YOUR_API_KEY_HERE>"
+server_config:
+    processor_config: 
+        type: "coin_flip"
+    chain_id: 2
+    indexer_grpc_data_service_address: "grpc.testnet.aptoslabs.com:443"
+    auth_token: "<API-KEY>"
+    postgres_connection_string: "postgresql://localhost:5432/<Database Name>"
+    # Optional. Start processor at starting_version
+    starting_version: 636585029
+    # Optional. Stop processor after ending_version.
+    ending_version: 636589723
 ```
 
 ### More customization with config.yaml
 
 However, if you'd like to customize things further, you can change some of the other fields.
 
 If you'd like to start at a specific version, you can specify that in the config.yaml file with:
-
 ```yaml
-starting_version_default: 123456789
+starting_version: <Starting Version>
 ```
 
-This is the transaction version the indexer starts looking for events at. If the indexer has already processed transactions past this version, **it will skip all of them and go to the latest version stored.**
-
-The rows in `next_versions_to_process` are the `indexer_name` as the primary key and the `next_version` to process field, along with the `updated_at`.
+This is the transaction version the indexer starts looking for events at.
 
-If you want to **force** the indexer to backfill data (overwrite/rewrite data) from previous versions even though it's already indexed past it, you can specify this in the config.yaml file with:
+The rows in table `next_versions_to_process` are the `indexer_name` as the primary key and the `next_version` to process field, along with the `updated_at`.
 
 ```yaml
-starting_version_backfill: 123456789
+ending_version: <Ending Version>
 ```
 
-If you want to use a different network, change the `grpc_data_stream_endpoint` field to the corresponding desired value:
+If you'd would like to see a list of all Coin Flip transactions, you can search them in the [Aptos Explorer](https://explorer.aptoslabs.com/?network=testnet). Search for account `0xe57752173bc7c57e9b61c84895a75e53cd7c0ef0855acd81d31cb39b0e87e1d0` - which is the account where the CoinFlip module deployed to. In the result, you will see a list of transactions along with their version numbers. You can use the version numbers to specify the `starting_version` and `ending_version` in the config.yaml file.
+
+If you want to use a different network, change the `indexer_grpc_data_service_address` field to the corresponding desired value:
 
 ```yaml
-devnet: 35.225.218.95:50051
-testnet: 35.223.137.149:50051  # north america
-testnet: 34.64.252.224:50051   # asia
-mainnet: 34.30.218.153:50051
+grpc.devnet.aptoslabs.com:443
+grpc.testnet.aptoslabs.com:443
+grpc.mainnet.aptoslabs.com:443
```

Comment:
Does it need the `devnet: testnet: mainnet:` before each line?

Or at least a comment that says `#devnet` above each one